packages/wdio-dot-reporter/tests/index.test.ts

```diff
@@ -7,18 +7,19 @@ describe('Dot Reporter', () => {
         const logFile = tmp.fileSync()
         const reporter = new DotReporter({ logFile: logFile.name })
         reporter.write = jest.fn()
+        reporter.outputStream.write = jest.fn()
 
         reporter.onTestSkip()
-        expect(reporter.write.mock.calls[0][0]).toBe('cyanBright .')
-        reporter.write.mockClear()
+        expect((reporter.write as jest.Mock).mock.calls[0][0]).toBe('cyanBright .');
```

Comment:
Can you defined a variable for `reporter.write as jest.Mock` to avoid using `as` that many times

Reply:
Changes done